views/log.html

```diff
@@ -0,0 +1,3 @@
+<h1>Hello World Puddle</h1>
```

Comment:
This should be full HTML as it is not a partial.

So you'll want <html>, <head>, <body> etc